lib/builder.js

```diff
@@ -122,7 +123,7 @@ function writeOutputFile(opts, output, basePath) {
   });
 }
 
-exports.inlineSourceMap = function(output) {
+var inlineSourceMaps = exports.inlineSourceMap = function(output) {
```

Comment:
It's a tricky one. We're not a transpiler like Babel that deals in a single source - we work on sources. So I think it makes sense to think in the plural for operations. I think I'm happy with `opts.sourceMaps`, but let me know if you think it's confusing? I was also referring to being consistent with `inlineSourceMaps` and `inlineSourceMap` both being used here.